test/e2e/page-objects/pages/home/bitcoin-homepage.ts

```diff
@@ -0,0 +1,112 @@
+import HomePage from './homepage';
+
+class BitcoinHomepage extends HomePage {
+  protected readonly balance =
+    '[data-testid="coin-overview__primary-currency"]';
+
+  private readonly bridgeButton = {
+    text: 'Bridge',
+    tag: 'button',
+  };
+
+  private readonly buySellButton = '[data-testid="coin-overview-buy"]';
```

Comment:
what do you think on doing this with test ids, just to improve a bit readability, but non-blocking

```suggestion
  private readonly buySellButton = {
    testId: 'coin-overview-buy',
  }
```

Reply:
As mentioned in the comment below, we will keep it like this for now.

Additionally, I think we have not standardized the use of `testId` over `data-testid`. Personally, I am not entirely convinced that using `testId` is always better than `[data-testid="xxx"]`;. When copy paste from the element inspector or searching in the inspector, using  `[data-testid="xxx"]` is easier because we don't need to tweak anything the format, we can just copy paste the whole string.
For readability, since we have Ids as selectors in the inspector, I also believe `data-testid` provides more direct and explicit readability. I think both approaches are valid. We can use both based on personal preference until we standardize

Reply:
sounds good to me!